HW06.md

```diff
@@ -0,0 +1,1131 @@
+    ##   NumReads = col_double()
+    ## )
+    ## Parsed with column specification:
+    ## cols(
+    ##   Name = col_character(),
+    ##   Length = col_double(),
+    ##   EffectiveLength = col_double(),
+    ##   TPM = col_double(),
+    ##   NumReads = col_double()
+    ## )
+    ## Parsed with column specification:
+    ## cols(
+    ##   Name = col_character(),
+    ##   Length = col_double(),
+    ##   EffectiveLength = col_double(),
+    ##   TPM = col_double(),
+    ##   NumReads = col_double()
+    ## )
+    ## Parsed with column specification:
+    ## cols(
+    ##   Name = col_character(),
+    ##   Length = col_double(),
+    ##   EffectiveLength = col_double(),
+    ##   TPM = col_double(),
+    ##   NumReads = col_double()
+    ## )
+    ## Parsed with column specification:
+    ## cols(
+    ##   Name = col_character(),
+    ##   Length = col_double(),
+    ##   EffectiveLength = col_double(),
+    ##   TPM = col_double(),
+    ##   NumReads = col_double()
+    ## )
+    ## Parsed with column specification:
+    ## cols(
+    ##   Name = col_character(),
+    ##   Length = col_double(),
+    ##   EffectiveLength = col_double(),
+    ##   TPM = col_double(),
+    ##   NumReads = col_double()
+    ## )
+    ## Parsed with column specification:
+    ## cols(
+    ##   Name = col_character(),
+    ##   Length = col_double(),
+    ##   EffectiveLength = col_double(),
+    ##   TPM = col_double(),
+    ##   NumReads = col_double()
+    ## )
+    ## Parsed with column specification:
+    ## cols(
+    ##   Name = col_character(),
+    ##   Length = col_double(),
+    ##   EffectiveLength = col_double(),
+    ##   TPM = col_double(),
+    ##   NumReads = col_double()
+    ## )
+    ## Parsed with column specification:
+    ## cols(
+    ##   Name = col_character(),
+    ##   Length = col_double(),
+    ##   EffectiveLength = col_double(),
+    ##   TPM = col_double(),
+    ##   NumReads = col_double()
+    ## )
+
+``` r
+library(tidyverse)
+```
+
+    ## ── Attaching packages ───────────────────────────────────────────────────── tidyverse 1.3.0 ──
+
+    ## ✓ ggplot2 3.3.2.9000     ✓ dplyr   1.0.1     
+    ## ✓ tibble  3.0.3          ✓ stringr 1.4.0     
+    ## ✓ tidyr   1.1.1          ✓ forcats 0.5.0     
+    ## ✓ purrr   0.3.4
+
+    ## ── Conflicts ──────────────────────────────────────────────────────── tidyverse_conflicts() ──
+    ## x dplyr::filter() masks stats::filter()
+    ## x dplyr::lag()    masks stats::lag()
+
+``` r
+library(magrittr)
+```
+
+    ## 
+    ## Attaching package: 'magrittr'
+
+    ## The following object is masked from 'package:purrr':
+    ## 
+    ##     set_names
+
+    ## The following object is masked from 'package:tidyr':
+    ## 
+    ##     extract
+
+``` r
+library(dplyr)
+
+#add type column to distinguish WT from mutated
+WT_df <- WT_df %>% mutate("type" = "WT")
+H3K27M_df <- H3K27M_df %>% mutate("type" = "H3K27_Mutated")
+```
+
+``` r
+#join into single dataframe for subsequent analysis
```

Comment:
This is definitely the more efficient way of doing this. I kept my H3K27M and WT data in separate lists for a while, but I only realized how inefficient that was until I was halfway done and didn't want to run the risk of messing up the code by going back to fix it.